app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -90,7 +105,50 @@ class ExportWorkflowsDialog extends React.Component {
 ExportWorkflowsDialog.propTypes = {
   project: React.PropTypes.shape({ links: React.PropTypes.object }).isRequired,
   onSuccess: React.PropTypes.func.isRequired,
-  onFail: React.PropTypes.func.isRequired,
+  onFail: React.PropTypes.func.isRequired
+};
+
+const ExportWorkflowListItem = ({ workflow, media, onChange }) => {
+  const myMedia = workflow ? media[workflow.id.toString()] : null;
+  const now = new Date();
+  const lockoutTime = new Date();
+  lockoutTime.setDate(now.getDate() - 1);
+
+  const lockout = myMedia && (new Date(myMedia.updated_at) > lockoutTime);
+  const titleString = lockout ? 'This item can only be exported every 24 hours' : '';
+
+  return (
+    <li>
+      <input type="radio" title={titleString} name="which-workflow" id={`export-${workflow.id}`} disabled={lockout} onChange={onChange} />
+      {workflow.display_name}
+      <small>
+        <ExportWorkflowLink media={myMedia} />
+      </small>
+    </li>
+  );
+};
+
+ExportWorkflowListItem.propTypes = {
+  workflow: React.PropTypes.shape({ id: React.PropTypes.string, display_name: React.PropTypes.string }).isRequired,
+  media: React.PropTypes.shape({}).isRequired,
```

Comment:
We don't use any parts of the shape in this component, because we just pass it on to the child component. The child component validates the props that it cares about, so it would probably be redundant to do it here. The linter rules don't let us simply specify that it's an object and is required, so this is the best solution we could think of.